api/overview.adoc

```diff
@@ -4,15 +4,15 @@
 :toc:
 ----
 
-Tendrl provides a RESTful APIs accessible via HTTP(s).
```

Comment:
I think change from **RESTful API** to only **API** is here because of https://github.com/Tendrl/documentation/issues/70 , right?